whois-commons/src/main/java/net/ripe/db/whois/common/DatabaseDummifierJmx.java

```diff
@@ -43,33 +42,34 @@
 @ManagedResource(objectName = JmxBase.OBJECT_NAME_BASE + "Dummifier", description = "Whois data dummifier")
 /**
  * in jmxterm, run with:
- *      run dummify jdbc:mariadb://<host>/<db> <user> <pass> <env>
+ *      run dummify
  *
  * in console, run with
- *      java -Xmx1G -cp whois.jar net.ripe.db.whois.common.DatabaseDummifierJmx --jdbc-url jdbc:mariadb://localhost/BLAH --user XXX --pass XXX --env XXX
+ *      java -Xmx1G -cp whois.jar net.ripe.db.whois.common.DatabaseDummifierJmx
  *
  */
 public class DatabaseDummifierJmx extends JmxBase {
     private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseDummifierJmx.class);
-
-    private static final String ARG_JDBCURL = "jdbc-url";
-    private static final String ARG_USER = "user";
-    private static final String ARG_PASS = "pass";
-
-    private static final String ARG_ENV = "env";
-
     private static TransactionTemplate transactionTemplate;
-    private static JdbcTemplate jdbcTemplate;
 
-    private static JdbcTemplate internalTemplate;
+    private final Environment environment;
+    private final JdbcTemplate jdbcTemplate;
 
     private static final DummifierRC dummifier = new DummifierRC();
-
     private static final AtomicInteger jobsAdded = new AtomicInteger();
     private static final AtomicInteger jobsDone = new AtomicInteger();
 
-    public DatabaseDummifierJmx() {
+    @Autowired
+    public DatabaseDummifierJmx(@Value("${whois.environment:}") final String environment,
```

Comment:
Should the environment be mandatory? Or default to e.g. localhost ?

Reply:
I think that it should be mandatory, rather than adding a localhost